pybossa/api/task.py

```diff
@@ -72,6 +72,8 @@ def _update_attribute(self, new, old):
                 new.state = 'completed'
         new.calibration = int(gold_task)
         new.exported = gold_task
+        current_app.logger.info("Task %d updated, new state: %s, new exported: %s",
```

Comment:
This is the place where attributes are set for task update; task hasn't been updated yet. Would it be useful to modify log statement and also record old states. Something like below or similar to it
```python
current_app.logger.info("Task %d updating attributes, state from %s to %s , exported from %s to %s",
      new.id, old.state, str(old.exported), new.state, str(new.exported))
```